.gitignore

```diff
@@ -105,8 +105,8 @@ ipython_config.py
 #   This is especially recommended for binary packages to ensure reproducibility, and is more
 #   commonly ignored for libraries.
 #   https://python-poetry.org/docs/basic-usage/#commit-your-poetrylock-file-to-version-control
-#poetry.lock
-#poetry.toml
+poetry.lock
+poetry.toml
```

Comment:
Is this ignore wanted?

Reply:
It was intended. 
We are not currently committing lock files.
We could consider adding a `poetry.toml`, but I didn't need it.
What version of poetry do you have?